packages/cashier/src/hooks/useCountdown.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react';
+
+const ONE_SECOND = 1000;
```

Comment:
Do you think it's a good idea to move these constants to `constants.js`? Maybe we can reuse it elsewhere if required 🤔

Reply:
@nijil-binary I think that would be creating unnecessary dependencies which causes jumping between files while maintaining the code in the future, But if that’s our code conversation I can move it to `constants.js` 🙇🏻  Let me know if I should make the changes 🙏🏻

Reply:
Was a NIT comment. If you think it's not necessary then LGTM 👌